### PR TITLE
Added release notes for teleport pointer improvements added teleport completed sound

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Added release notes for the teleport pointer improvements associated with #8259 and #8386

### CODE REVIEW

**Review by keveleigh — APPROVED** [review]

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ MRTK now supports running Oculus Quest Headsets and Controllers using the native
 
 For instructions on how to deploy your device on the Oculus Quest using the new pipeline, see the [Oculus Quest Setup Guide](CrossPlatform/OculusQuestMRTK.md)
 
+**Teleport pointer animation, handling, and sound improvements**
+
+The teleport pointer now has improved animations and audio feedback. We also improved the handling of the teleport pointer so it handles smoother when transitioning from pointing at nearby surfaces to farther away surfaces.
```
Comment (wiwei) [inline Documentation/ReleaseNotes.md:27]: Extra credit here and also fine for a future thing, but if you have a video or gif to share that would be really neat

Comment (RogPodge) [reply Documentation/ReleaseNotes.md:27]: are streamable links fine like this one? https://streamable.com/ta4bhv

**Pull request merged**